src/main/java/org/folio/linked/data/client/SpecClient.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.linked.data.client;
+
+import static org.folio.linked.data.util.Constants.Cache.SPEC_RULES;
+
+import java.util.UUID;
+import org.folio.rspec.domain.dto.SpecificationDtoCollection;
+import org.folio.rspec.domain.dto.SpecificationRuleDtoCollection;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+@FeignClient(name = "specification-storage")
+public interface SpecClient {
+
+  @Cacheable(cacheNames = "bib-marc-specs")
```

Comment:
Looks like this cache is not evicted.

Reply:
yes, but for this one it's not necessary is it?